pyproject.toml

```diff
@@ -21,8 +21,8 @@ packages = [{ include = "qibo_cloud_backends", from = "src" }]
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-qibo = ">=0.2.5"
-qibo_client = ">=0.0.2"
+qibo = "==0.2.4"
```

Comment:
Should we keep this?

If higher versions are not allowed (it has to be the exact same one, client and server) this should be imposed by `qibo_client` here
https://github.com/qiboteam/qibo-client/blob/4e70c049bb66475d9af468fe3e9611d75fd5e5fb/pyproject.toml#L24
not by the cloud backends.

Reply:
With qibo-client 0.0.3, it is now possible to use a local qibo version `>=` than the server.

Reply:
Applied in a34ef02